src/wikimedia/wikiFetch.ts

```
@@ -3,11 +3,28 @@ import nextConfig from "../../next.config";
 import { memoizeDevServerConst } from "../utils/memoizeDevServerConst";
 import type { PapyrusWiki } from "./getWiki";
 
-const wikiFetchPromisesByURL = memoizeDevServerConst('wikiFetchCache', ()=>new Map<string, Promise<{}|null>>());
+const wikiFetchPromisesByURL = memoizeDevServerConst('wikiFetchCache', ()=>{
+    const map = new Map<string, Promise<{}|null|typeof WIKI_FETCH_403FORBIDDEN>>();
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)